src/pivot-table/data/extract-headers.ts

```diff
@@ -53,21 +44,21 @@ const createHeaderCell = (
   }
   return {
     id,
-    colIdx,
+    colIdx: 0, // has to be properly set later, would be incorrect after transposing otherwise
```

Comment:
If this is set to `0` here, does that mean that cells at last column but not at the bottom row are all getting `colIdx === 0`? Since only the last row is reassigned later?

Reply:
yeah that is what I try to explain in the PR description. This value was incorrect before, both for the header menu and the column adjuster. Now it is correct for the column adjuster at least. This needs another look